process/README.md

```diff
@@ -9,64 +9,56 @@ The process check lets you:
+          ## @param name - string - required
+          ## Used to uniquely identify your metrics
+          ## as they are tagged with this name in Datadog.
+          #
+        - name: ssh
 
-You can also configure the check to find any process by exact PID (`pid`) or pidfile (`pid_file`). If you provide more than one of `search_string`, `pid`, and `pid_file`, the check uses the first option it finds in that order (e.g. it uses `search_string` over `pid_file` if you configure both).
+          ## @param search_string - list of strings - required
+          ## If one of the elements in the list matches, it return the count of
+          ## all the processes that match the string exactly by default.
+          ## Change this behavior with the parameter `exact_match: false`.
+          #
+          search_string: ['ssh', 'sshd']
+    ```
 
-**Note** For agent v6.11+ on windows, the agent runs as an unprivileged `ddagentuser`, and does not have access to the full command line of processes running under a different users, so the `exact_match: false` option cannot be used in such cases. The same goes for the `user` option that allows you to select only processes belonging to a specific user.
-
-To have the check search for processes in a path other than `/proc`, set `procfs_path: <your_proc_path>` in `datadog.conf`, NOT in `process.yaml` (its use has been deprecated there). Set this to `/host/proc` if you're running the Agent from a Docker container (i.e. [docker-dd-agent][4]) and want to monitor processes running on the server hosting your containers. You DON'T need to set this to monitor processes running _in_ your containers; the [Docker check][5] monitors those.
-
-Some process metrics require either running the datadog collector as the same user as the monitored process or privileged access to be retrieved.
-Where the former option is not desired, and to avoid running the datadog collector as `root`, the `try_sudo` option lets the process check try using `sudo` to collect this metric.
-As of now, only the `open_file_descriptors` metric on Unix platforms is taking advantage of this setting.
-Note: the appropriate sudoers rules have to be configured for this to work
-```
-dd-agent ALL=NOPASSWD: /bin/ls /proc/*/fd/
-```
+    Some process metrics require either running the datadog collector as the same user as the monitored process or privileged access to be retrieved. Where the former option is not desired, and to avoid running the datadog collector as `root`, the `try_sudo` option lets the process check try using `sudo` to collect this metric. As of now, only the `open_file_descriptors` metric on Unix platforms is taking advantage of this setting. Note: the appropriate sudoers rules have to be configured for this to work:
```

Comment:
```suggestion
    Some process metrics require either running the Datadog collector as the same user as the monitored process or privileged access to be retrieved. Where the former option is not desired, and to avoid running the Datadog collector as `root`, the `try_sudo` option lets the process check try using `sudo` to collect this metric. As of now, only the `open_file_descriptors` metric on Unix platforms is taking advantage of this setting. Note: the appropriate sudoers rules have to be configured for this to work:
```

Reply:
well spotted !!!!